plugins/node/opentelemetry-instrumentation-cassandra/test/cassandra-driver.test.ts

```diff
@@ -228,6 +246,60 @@ describe('CassandraDriverInstrumentation', () => {
         assertSingleSpan('cassandra-driver.execute', query.substr(0, 25));
       });
     });
+
+    describe('responseHook', () => {
+      after(() => {
+        instrumentation.setConfig({});
+      });
+
+      it('adds custom attributes to span', async () => {
+        const responseAttributeName = 'response.attribute';
+        const customAttributeName = 'custom.attribute';
+        const customAttributeValue = 'custom attribute value';
+
+        const config: CassandraDriverInstrumentationConfig = {
+          responseHook: (span: Span, responseInfo: ResponseInfo) => {
+            const row = responseInfo.response.first();
+            const responseValue = parseInt(row.count);
```

Comment:
according to the docs, [`parseInt` argument is `string`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt).

If `row.count` is [long](https://www.npmjs.com/package/long), might be more correct to use `row.count.toNumber()`?

Reply:
It does parse it correctly, but yes this makes more sense :)